win32ss/user/ntuser/ime.c

```diff
@@ -224,9 +224,12 @@ AllocInputContextObject(PDESKTOP pDesk,
 VOID UserFreeInputContext(PVOID Object)
 {
     PIMC pIMC = Object, pImc0;
-    PTHREADINFO pti = pIMC->head.pti;
+    PTHREADINFO pti;
 
-    UserMarkObjectDestroy(Object);
+    if (!pIMC)
+        return;
+
+    pti = pIMC->head.pti;
 
     for (pImc0 = pti->spDefaultImc; pImc0; pImc0 = pImc0->pImcNext)
```

Comment:
Missing comment: `/* Find the IMC in the list and remove it */`

Reply:
OK.